README.md

```
@@ -1,5 +1,8 @@
 # s3rm
 
+[![Crates.io](https://img.shields.io/crates/v/s3rm-rs.svg)](https://crates.io/crates/s3rm-rs)
+![](https://img.shields.io/crates/d/s3rm-rs?label=downloads%20%28crates.io%29)
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)